### PR TITLE
chungthuang: Skip build when container starts, lower staging resource…

### PR DESCRIPTION
**Pull request by chungthuang**
… limit

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,9 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next build && next start",
+    "start": "next start",
+    "test": "react-scripts test --env=jsdom",
```
Comment (Digital-Coder) [inline package.json:57]: react-scripts does not exist in next.js, needs to be removed

File: package.json [hunk 2]
```diff
@@ -53,7 +53,9 @@
...
+    "eject": "react-scripts eject",
```
Comment (Digital-Coder) [inline package.json:58]: react-scripts does not exist in next.js and needs to be removed

Comment (chungthuang) [reply package.json]: Great catch, I accidentally introduced them in a rebase conflict.

**Pull request merged**